crates/ruff_python_parser/src/parser/mod.rs

```diff
@@ -66,9 +66,24 @@ bitflags! {
     }
 }
 
-type ExprWithRange = (Expr, TextRange);
+type ExprWithRange = (ParsedExpr, TextRange);
 type StmtWithRange = (Stmt, TextRange);
 
+#[derive(Debug)]
+struct ParsedExpr {
+    expr: Expr,
+    is_parenthesized: bool,
+}
```

Comment:
Is it possible to treat this as an `Expr` so that I don't have to type `.expr` every time?

Reply:
You can implement Deref on `ParsedExpr`

Reply:
Oh right, forgot about Deref.